### PR TITLE
e2e for java, changes to pom, new iot

### PR DESCRIPTION
**Pull request by BartMoroz**

### CODE REVIEW

File: sdk/pom.xml [hunk 1]
```diff
@@ -74,9 +76,14 @@
 		<dependency>
 			<groupId>junit</groupId>
 			<artifactId>junit</artifactId>
-			<version>4.4</version>
+			<version>4.12</version>
```
Comment (maciej-wakula-mobica) [inline sdk/pom.xml]: Why downgrading the version?

Comment (maciej-wakula-mobica) [reply sdk/pom.xml]: 412. is newer than 4.4....

File: sdk/src/main/resources/testcfg.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+		"expMonth": 12,
+		"expYear": 2020,
+		"cardNumber": "34343434343434",
+		"type": "Card",
+		"cvc": "123"
+	},
+	"producerName": "e2e-test",
+	"producerDescription": "test-producer",
+	"serviceID": 1,
+	"serviceName": "RoboWash",
```
Comment (maciej-wakula-mobica) [inline sdk/src/main/resources/testcfg.json]: Why RoboWash ?!?

Comment (BartMoroz) [reply sdk/src/main/resources/testcfg.json]: This is just for e2e test 🗡

**Pull request merged**